### PR TITLE
feat: Add python helper for formatting bytes

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

**Comment by dcramer** [comment]
This is needed for some code on sentry.io

File: src/sentry/utils/numbers.py [hunk 1]
```diff
@@ -62,3 +62,17 @@ def base36_encode(number):
...
+    block = 1024.0
+    if number < block:
+        return '{} {}'.format(number, units[0])
+
+    u = 0
+    max_unit = len(units) - 1
+    while number >= block and u < max_unit:
+        number /= block
+        u += 1
+    return (u'{:.%df} {}' % (decimal_places,)).format(number, units[u])
```
Comment (dcramer) [inline src/sentry/utils/numbers.py]: lazy.jpg

**Review by mattrobenolt — APPROVED** [review]
These are great gibibytes.

File: src/sentry/utils/numbers.py [hunk 2]
```diff
@@ -62,3 +62,17 @@ def base36_encode(number):
 
 def base36_decode(s):
     return int(s, 36)
+
+
+def format_bytes(number, units=['B', 'KB', 'MB', 'GB', 'TB',
+                                'PB', 'EB', 'ZB', 'YB'], decimal_places=2):
```
Comment (markstory) [inline src/sentry/utils/numbers.py]: Will units ever be used as argument? You have a mutable type as a default arg which doesn't work great.

Comment (dcramer) [reply src/sentry/utils/numbers.py]: micro ptimization -- it only matters if you were gonna mutate it, but ill fix, and yes it'll be used

**Pull request merged**